resources/sagemaker_endpoint_configurations.go

```diff
@@ -0,0 +1,201 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/sagemaker"
+
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func SagemakerEndpointConfigurations() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_sagemaker_endpoint_configurations",
+		Description:  "Provides summary information for an endpoint configuration.",
+		Resolver:     fetchSagemakerEndpointConfigurations,
+		Multiplex:    client.AccountRegionMultiplex,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"arn"}},
+		Columns: []schema.Column{
+			{
+				Name:        "account_id",
+				Description: "The AWS Account ID of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSAccount,
+			},
+			{
+				Name:        "region",
+				Description: "The AWS Region of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSRegion,
+			},
+			{
+				Name:        "kms_key_id",
+				Description: "Amazon Web Services KMS key ID Amazon SageMaker uses to encrypt data when storing it on the ML storage volume attached to the instance.",
+				Type:        schema.TypeString,
+			},
+			{
```

Comment:
missing description

Reply:
https://github.com/aws/aws-sdk-go-v2/blob/56f090f03626e50ab996bd0e8a80ef13f7ab85fc/service/sagemaker/api_op_DescribeEndpointConfig.go#L71

Missing information on this one